README.md

```diff
@@ -39,6 +39,10 @@ for the database, and run `ironblogger init-db`
 
 Static assets have been submoduled and are kept in `ironblogger/static/bower_components`, which used to be managed by Bower, which is why it has that name.
 
+To pull these in, run `git submodule init` then `git submodule update --depth=1` this will dowload all the required css and other media files.
```

Comment:
1. This should be two sentences; you're missing a period after the second command.
2. Please hard-wrap the lines in this file to the usual 72-80 chars.